code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -135,6 +135,9 @@ GLOBAL_LIST_INIT(metal_recipes, list(
 	return "Use in your hand to bring up the recipe menu. If you have enough sheets, click on something on the list to build it.<br>\
 			You can replenish your supply of metal as a synthetic by recharging."
 
+/obj/item/stack/sheet/metal/five
```

Comment:
not a massive fan of this, you can just specify the amount with the new constructor

Reply:
should be handled properly now